### PR TITLE
Copy FigmaColor to dc_bundle as FloatColor

### PR DESCRIPTION
**Pull request by yiqunw700**

### CODE REVIEW

File: crates/figma_import/src/reflection.rs [hunk 1]
```diff
@@ -177,6 +177,9 @@ pub fn registry() -> serde_reflection::Result<serde_reflection::Registry> {
     tracer
```
Comment (yiqunw700) [inline crates/figma_import/src/reflection.rs:177]: Let me know if I should remove it or keep it..
For the figma_schema::FigmaColor. I am not quite sure why some need in reflection and some not.

Comment (timothyfroehlich) [reply crates/figma_import/src/reflection.rs:177]: Yes, remove the FigmaColor one. Reflection basically defines the types that the serde code generates Java code for. Primarily that is the types used in DesignDefinition. The goal is to keep FigmaColor out of DesignDefinition so it should be removed from reflection.rs.

**Review by timothyfroehlich — CHANGES_REQUESTED** [review]

File: crates/figma_import/src/reflection.rs (hunk 1 above)
Comment (yiqunw700) [reply crates/figma_import/src/reflection.rs:177]: Done.

File: crates/figma_import/src/variable_utils.rs [hunk 2]
```diff
@@ -70,7 +71,9 @@ fn create_variable_value(v: &figma_schema::VariableValue) -> VariableValue {
         figma_schema::VariableValue::Boolean(b) => VariableValue::Bool(b.clone()),
         figma_schema::VariableValue::Float(f) => VariableValue::Number(f.clone()),
         figma_schema::VariableValue::String(s) => VariableValue::Text(s.clone()),
-        figma_schema::VariableValue::Color(c) => VariableValue::Color(c.into()),
+        figma_schema::VariableValue::Color(c) => VariableValue::Color(
+            (&(<&figma_schema::FigmaColor as Into<FloatColor>>::into(c))).into(),
+        ),
```
Comment (rylin8) [inline crates/figma_import/src/variable_utils.rs:76]: This goes FigmaColor -> FloatColor -> Color ? Can we just keep the implementation of Into<Color> for &FigmaColor ?

Comment (yiqunw700) [reply crates/figma_import/src/variable_utils.rs]: Done.

**Review by timothyfroehlich — APPROVED** [review]

**Review by rylin8 — APPROVED** [review]

**Pull request merged**